src/mcp_jupyter/server.py

```diff
@@ -559,12 +672,24 @@ def modify_notebook_cells(
     position_index: int = None,
     execute: bool = True,
 ) -> dict:
-    """Modify notebook cells (add, edit, delete) on the user-provided server.
+    r"""Modify notebook cells (add, edit, delete) on the user-provided server.
```

Comment:
Is there a reason to change this to a raw string?

Reply:
Ruff started wanting that, but I can adjust ruff.

Reply:
https://peps.python.org/pep-0257/#what-is-a-docstring